File: day 6/review.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int n;
    cin >> n;
    string input;
    
    for (int i = 0; i < n; i++){
        cin >> input;
        string odd,even ="";
        for (int j = 0; j < input.length(); j++){
            if (j%2==1)
                odd+=input[j];
            else
                even+=input[j];
        }
        cout << even << " " << odd << endl;
    }

    return 0;
}

File: day 6/review.cs
using System;
using System.Collections.Generic;
using System.IO;
class Solution {
    static void Main(String[] args) {
        int n;
        n = Convert.ToInt32(Console.ReadLine());

        for (int i = 0; i < n; i++){
            string input;
            input = Console.ReadLine();
            string even = "";
            string odd = "";
            for (int j = 0; j < input.Length; j++)
            {
                if (j%2==1){
                    odd+=input[j];
                } else {
                    even+=input[j];
                }
            }
            Console.WriteLine(even+" "+odd);
        }
    }
}


